src/views/Home/components/DiningBalance/index.jsx

```diff
@@ -138,7 +138,7 @@ const DiningBalance = () => {
           style={{ paddingTop: 5, paddingBottom: 10 }}
         >
           <Grid item>
-            <Typography variant="body2" className={styles.label2}>
+            <Typography variant="body2" style={{ color: 'gray', textAlign: 'center' }}>
```

Comment:
I've seen a lot of changes like this.  Is it really better?  The old code seems cleaner, so I wonder what's driving the change?